metatron-app/metatron-discovery
# F#923 fix settype editor

### Description

There was a problem with the textinput focus and modified it.

**Related Issue** : 

#923 

### How Has This Been Tested?

1. menu > MANAGEMENT > DataPreparation > Dataflow
2. click Dataflow > Click Wrangled Data > Edit Rule
3. Add Rule
4. Command : Settype > Column : numeric type column > New type : timestamp
5. See "Set pattern"

### Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)


### Checklist:



### Additional Context
